torch/distributed/fsdp/_optim_utils.py

```diff
@@ -493,6 +490,7 @@ def _flatten_optim_state_dict(
                     else:
                         # Move the tensor in the original osd back to CPU to make the
                         # original osd unaffected.
+                        continue
```

Comment:
QQ about the continue: Are we just skipping moving the state back to CPU here?

Reply:
Oop, sorry, didn't remove this before submitting the PR. Will update it.